space-plugins/nuxt-base-ui/components/BaseModal.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+defineProps<{
+	title?: string;
+	message?: string;
+}>();
+
+const modalRef = ref<HTMLDialogElement | null>(null);
+
+const show = () => modalRef.value?.showModal();
+const close = () => modalRef.value?.close();
+
+defineExpose({
+	show,
+	close,
+});
+</script>
+
+<template>
+	<dialog ref="modalRef" class="modal">
+		<form method="dialog" class="modal-box p-0 rounded-lg">
+			<section class="flex flex-col gap-4 p-9 items-center">
+				<h2 v-if="title" class="font-bold text-xl">
+					{{ title }}
+				</h2>
+				<p v-if="message" class="py-4 text-ink-50 font-normal">
+					{{ message }}
```

Comment:
nit pick: no need to implement it, especially for the first version everything looks already amazing. Just wondering if in future {{message}} should also turn into a slot, in case the user wants to use the modal with input fields or custom components instead of string-only. What do you think? :)